guides/v2.2/config-guide/redis/redis-pg-cache.md

```diff
@@ -27,21 +29,25 @@ where
 |--- |--- |--- |--- |
 |cache-backend-redis-server|server|Fully qualified hostname, IP address, or an absolute path to a UNIX socket. The default value of 127.0.0.1 indicates Redis is installed on the Magento server.|127.0.0.1|
 |cache-backend-redis-port|port|Redis server listen port|6379|
-|cache-backend-redis-db|database|Required if you use Redis for both the default and full page cache. You must specify the database number of one of the caches; the other cache uses 0 by default.
-Important: If you use Redis for more than one type of caching, the database numbers must be different. It is recommended that you assign the default caching database number to 0, the page caching database number to 1, and the session storage database number to 2.|0|
+|cache-backend-redis-db|database|Required if you use Redis for both the default and full page cache. You must specify the database number of one of the caches; the other cache uses 0 by default.<br/><br/>Important: If you use Redis for more than one type of caching, the database numbers must be different. It is recommended that you assign the default caching database number to 0, the page caching database number to 1, and the session storage database number to 2.|0|
 |cache-backend-redis-password|password|Configuring a Redis password enables one of its built-in security features: the auth command, which requires clients to authenticate to access the database. The password is configured directly in Redis's configuration file, /etc/redis/redis.conf, which you should still have open from the previous step.||
+{:style="table-layout:auto;"}
 
 ### Example command
 
-The following example enables Redis default caching, sets the host to `127.0.0.1` and assigns the database number to 0. All other parameters are set to the default value.
+The following example enables Redis default caching, sets the host to `127.0.0.1` and assigns the database number to 0. Redis uses default values for all other parameters.
 
+```bash
 `bin/magento setup:config:set --cache-backend=redis --cache-backend-redis-server=127.0.0.1 --cache-backend-redis-db=0`
+```
 
 ## Configure Redis page caching
 
 To configure Redis page caching on Magento, run the `setup:config:set` command with additional parameters.
 
+```bash
 `bin/magento setup:config:set --page-cache=redis --page-cache-redis-<parameter_name>=<parameter_value>...`
```

Comment:
Remove the single backticks surrounding the command.

Reply:
Done.